packages/fx-core/src/plugins/resource/sql/plugin.ts

```diff
@@ -217,6 +219,15 @@ export class SqlPluginImpl {
   }
 
   public async generateArmTemplates(ctx: PluginContext): Promise<Result<any, FxError>> {
+    TelemetryUtils.init(ctx.telemetryReporter);
```

Comment:
similar to another comment, it may be better to initialize telemetry util globally. Otherwise we may forgot init somewhere

Reply:
It is hard to initialize it as the plugin instance is created by core. As there is no context passed, we can't initialize telemetry reporter.
And the core can call every exposed method of plugin instance. We can only do this inside the method.

Reply:
ok， thanks